packages/websocket-client/tests/client.test.ts

```diff
@@ -95,6 +88,7 @@ describe('WebsocketClient', () => {
         jest.useFakeTimers();
 
         const cli = new Client({ url: server.getUrl(), pingTimeout: 5000 });
+        // @ts-expect-error ping is protected
         const pingSpy = jest.spyOn(cli, 'ping');
```

Comment:
ping is just sendMessage with payload { method: 'ping' }, and sendMessage is already public, so ping might be as well 🤷

Reply:
It might be, but I prefer not to, at least until I see legitimate reason for pinging from outside.